code/src/components/experience-type/experience-type.style.js

```diff
@@ -0,0 +1,11 @@
+import styled from "styled-components";
+
+const ClimbingTypeSelect = styled.select`
```

Comment:
Is this a typescript thing? Or a CSS thing? Looks like a good way of using variables and then style them in a way way👏

Reply:
This is styled-components actually a CSS thing.  Here you can read about some reasons for use it and also how to use it. https://styled-components.com/docs/basics#motivation